pyca/db.py

```diff
@@ -194,3 +196,17 @@ def __init__(self, service=None):
         if service:
             self.type = service.type
             self.status = service.status
+
+
+class UpstreamState(Base):
+    '''State of the upstream Opencast server.'''
+    __tablename__ = 'upstream_state'
+    url = Column('url', Text(), primary_key=True)
+    last_synced = Column('last_synced', DateTime())
+
+    @staticmethod
+    def update(url):
```

Comment:
Maybe to be a bit more clear what you update:
```suggestion
    def update_sync_time(url):
```